### DISCUSSION TITLE
Where to find the validator_dummy.properties?

### DISCUSSION THREAD

**Discussion by gregor-rieble** [Keyfactor/signserver-ce#15] [Q&A]
I'm trying to get familiar with sign server and now I'm having trouble to configure a working XML validator. I tried to follow the documentation, but the documentation talks about a `validator_dummy.properties` file:

[QuickStartDemoSetupusingAdministrationCLI-SetupaSampleXMLvalidator](https://doc.primekey.com/signserver510/signserver-operations/worker-setup/quick-start-demo-setup-using-administration-cli#QuickStartDemoSetupusingAdministrationCLI-SetupaSampleXMLvalidator)

However, this file does not exists. There is only a `validator.properties`,  but it seems to not be the correct one, because it does define 2 workers (OSCP/CRL) and does not define a worker named `CertValidationWorker` as it is stated in the documentation.

Can you please push me into the right direction on how to get a XML validator working with the dummy setup?

**Comment by netmackan** [comment]
The documentation seems a bit outdated here. With the new validator.properties file a certificate validator named CRLValidator (id 15) and one called OCSPValidator (id 16) are setup. This means for your XMLValidator you need to chose one of them and set VALIDATIONSERVICEWORKER to the name of that one.

To be honest I think the validation feature is rarely used. Especially this one which is rather complicated to configure. You might have better luck configuring an XAdESSigner and XAdESValidator. That validator should be more easy to setup with only two configuration parameters to set:
* The trusted certificates
* and if revocation checking should be used. A suggestion is to first turn this off and get that working before turning on revocation checking.

Cheers,
Markus

**Reply by netmackan** [reply]
Registered issue #16 for updating the documentation. Thanks for bringing this to our attention.

**Reply by 17Xamuel** [reply]
Thank you @netmackan.
I set up the CRLValidator from validator.properties template and referenced its id (15) in the properties of the XMLValidator worker. But whenever  I try to validate, it returns INVALID even if on an XML file I have just signed.
Then, is there any validator template for PDF  files, something like `pdfvalidator.properties`.
